include/cantera/zeroD/ReactorNet.h

```diff
@@ -12,6 +12,13 @@
 
 namespace Cantera
 {
+//! @defgroup zerodGroup Zero-Dimensional Reactor Networks
+//!
+//! @details See the
+//! [Reactor Science](https://cantera.org/science/reactors/reactors.html)
```

Comment:
This link is to the legacy website. To create a link that will stay within the correct section (stable or dev) of the website, this should be a relative link:
```suggestion
//! @details See the
//! [Reactor Science](../reference/reactors/index.html)
```
This is a bit tricky, but it is described at https://cantera.org/dev/develop/doc-formatting.html#useful-doxygen-syntax